src/mod/applications/mod_sms/mod_sms.c

```diff
@@ -610,6 +610,30 @@ SWITCH_STANDARD_CHAT_APP(reply_function)
 	return SWITCH_STATUS_SUCCESS;
 }
 
+
+SWITCH_STANDARD_CHAT_APP(transfer_function)
+{
+	char *context;
+
+	if (!data) return SWITCH_STATUS_SUCCESS;
+
+	context = strdup(data);
```

Comment:
Where is context freed?

Reply:
is not... my bad.  fixing it now.

thanks